test/integration/features/cmd-oc-env.feature

```diff
@@ -0,0 +1,31 @@
+@cmd-oc-env @command
+Feature: Command oc-env
+Command oc-env helps the user to set path to oc binary so she can run the oc command within the VM host.
+INFO: This feature runs against persistent shell instance. To use non-default shell, please
+select one from: bash, cmd, powershell, tcsh, zsh with TEST_WITH_BASH parameter of make integration.
+
+  Scenario: User starts shell instance without oc in PATH
+  INFO: This scenario starts interactive shell instance, which
+  will be closed in the end of this feature.
+    Given user starts shell instance
+     When executing in shell command "unset PATH"
+     Then executing in shell command "oc version" fails
```

Comment:
I feel we should use `oc status` instead of `oc version` as it actually shows the current OpenShift namespace/project and the cluster IP etc in the output

Reply:
@LalatenduMohanty Will take a look on that, thanks for suggestion.